app/models/concerns/profile_rules.rb

```diff
@@ -10,12 +10,12 @@ module ProfileRules
 
     def update_rules(ids: nil, ref_ids: nil)
       removed = ::ProfileRule.where(
-        rule_id: rule_ids_to_destroy(ids, ref_ids), profile_id: id
+        entity_id: rule_ids_to_destroy(ids, ref_ids), profile_id: id
```

Comment:
Doesn't this pick up the alias?

Reply:
Yeah, it does! Sorry, I forgot to revert this change